sbc_compassion/tools/import_letter_functions.py

```diff
@@ -485,9 +481,10 @@ def _find_languages(env, img, pattern_center, line_vals, test):
 
     # if pattern has not been detected
     if pattern_center is None:
-        return line_vals
+        return
 
-    template = line_vals['template_id']
+    template = env['sponsorship.correspondence.template'].browse(
```

Comment:
`line_vals['template_id']` is an integer value.
At line https://github.com/CompassionCH/compassion-modules/pull/155/files#diff-e41465c073f2badef6f5427249b3c86bR499
we use `template` object and need therefore to fetch the record.